src/apps/KintoApp.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721EnumerableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721BurnableUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/access/AccessControlUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/cryptography/ECDSAUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/Initializable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/utils/UUPSUpgradeable.sol";
+
+import "../interfaces/IKintoID.sol";
+import "../interfaces/IKintoApp.sol";
+import "../interfaces/IKintoWalletFactory.sol";
+
+// import "forge-std/console2.sol";
+
+/**
+ * @title KintoApp
+ * @dev A contract that holds all the information of a KintoApp
+ */
+contract KintoApp is
+    Initializable,
+    ERC721Upgradeable,
+    ERC721EnumerableUpgradeable,
+    ERC721BurnableUpgradeable,
+    AccessControlUpgradeable,
+    UUPSUpgradeable,
+    IKintoApp
+{
+    /* ============ Constants ============ */
```

Comment:
super nit: missing blank space

and what do you think about this style:
```suggestion
    // Constants -------------------------------------------------------------------
```

Reply:
I'll adjust the blank. Regarding the comments format, I like the other style better, seems just prettier to me xD